app/experimenter/templates/experiments/field_inline_help_link.html

```diff
@@ -0,0 +1,23 @@
+<div class="form-group row">
+  <div class="col-3 text-right">
+    <label class="col-form-label" for="{{ field.id_for_label }}">
+      <strong>{{ field.label }}</strong>
+      {% if not field.field.required %}
+        <span class="text-muted optional-marker">Optional</span>
+      {% endif %}
+    </label>
+    <br>
+    <a target="_blank" rel="noreferrer noopener" href="{{ field.help_text }}">Help</a>
```

Comment:
I'm a bit confused here -- is `field.help_text` a URL, or HTML? You seem to be using it as a URL here but HTML down below.

Reply:
Yea, that later piece was not needed. I think I was getting kinda some template fatigue going and I just missed taking that out. Thank you!!

Reply:
It's a URL. That was the point of this template... to have a input field with a help button that was a link, instead of popping up helper text underneath the box.